openml/runs/run.py

```diff
@@ -182,55 +182,59 @@ def _create_description_xml(self):
         description_xml = xmltodict.unparse(description, pretty=True)
         return description_xml
 
-def _parse_parameters(model, flow):
-    """Extracts all parameter settings from a model in OpenML format.
+    @staticmethod
```

Comment:
But it wasn't a method before, so I don't see why it became a method in the first place.